operating/manage-user-access/authentication-provider-structure.adoc

```diff
@@ -0,0 +1,86 @@
+
+[id="auth-provider-auth0-default-claim-mappings"]
+=== Auth0 default claim mappings
+The `Auth0` default claim mappings are the same as the OIDC default claim mappings.
+
+[id="auth-provider-saml20-default-claim-mappings"]
+=== SAML 2.0 default claim mappings
+The following list applies to SAML 2.0 default claim mappings:
+
+* `Subject.NameID` is mapped to `userid`
+* every SAML `AttributeStatement.Attribute` from the response gets mapped to its name
+
+[id="auth-provider-google-iap-default-claim-mappings"]
+=== Google IAP default claim mappings
+The following list provides the Google IAP default claim mappings:
+
+* `sub` to `userid`
+* `email` to `email`
+* `hd` to `hd`
+* `google.access_levels` to `access_levels`
+
+[id="auth-provider-userpki-default-claim-mappings"]
+=== User Certificates default claim mappings
+User certificates differ from all other authentication providers because instead of communicating with a third-party IDP, they get user information from certificates used by the user.
+
+The default claim mappings for user certificates include:
+
+* `CertFingerprint` to `userid`
+* `Subject -> Common Name` to `name`
+* `EmailAddresses` to `email`
+* `Subject -> Organizational Unit` to `groups`
+
+[id="auth-provider-openshift-auth-default-claim-mappings"]
+=== OpenShift Auth default claim mappings
+The following list provides the OpenShift Auth default claim mappings:
+
+* `groups` to `groups`
+* `uid` to `userid`
+* `name` to `name`
+
+[id="auth-provider-rules"]
+== Rules
+To authorize users, {product-title-short} relies on mapping rules that examine the values of specific claims such as `groups`, `email`, `userid`, and `name` from the user identity.
+Rules allow to map users who have attribute with a specific value to a specific role. Example of a rule: `key` is `email`, `value` is `john@redhat.com`, `role` is `Admin`.
+
+If the claim is absent, the mapping cannot succeed and the user does not get access to the required resources.
+Therefore, you need to ensure that the claims required to authorize users from your IDP (for example, `groups`) are included in the authentication response of your IDP to enable successful mapping.
+
+[id="auth-provider-minimum-access-role"]
+== Minimum access role
+The minimum access role is the role that all is assigned to every caller with an {product-title-short} token issued by a particular authentication provider. It is set to `None` by default.
+
+For example, if there is an authentication provider with the minimum access role of `Analyst`, all users who log in using this provider have the role of `Analyst` assigned to them.
+
+[id="auth-provider-required-attributes"]
+== Required attributes
+Required attributes can restrict issuing of the {product-title-short} token based on whether a user identity has an attribute with a specific value.
+For example, when `key` is `is_internal` and `value` is `true`, users with `is_internal` not set or set to `false` will not be logged in.
```

Comment:
May be:
```
Required attributes can restrict issuing of the {product-title-short} token based on whether a user identity has an attribute with a specific value.

For example, you can configure RHACS only to issue a token when the attribute `key` has the value `is_internal` and the attribute `value` has the value `true`. 
Users with the attribute `key` with the value `is_internal` and the attribute `value` with the value `false` do not get a token.
```

Reply:
Attributes(and claims) have key and then value(for example, attribute can have key `email` and value `john@redhat.com`). Tried to rewrite based on your suggestion